gridappsd/field_interface/gridappsd_field_bus.py

```diff
@@ -36,11 +36,26 @@ def subscribe(self, topic, callback):
     def unsubscribe(self, topic):
         pass
 
-    def publish(self, data, topic: str = None):
+    def send(self, topic, message):
         """
         Publish device specific data to the concrete message bus.
         """
-        pass
+        if self.gridappsd_obj is not None:
+            self.gridappsd_obj.send(topic, message)
+            
+    def get_response(self, topic, message, timeout=5):
```

Comment:
These two functions do the exact same thing.  Why should there be two of them?

Reply:
Removed duplicate. Will be good to have a get_agent_response but will add later.